api/src/main/java/org/openmrs/module/attachments/AttachmentsContext.java

```diff
@@ -189,6 +189,17 @@ public Encounter getAttachmentEncounter(Patient patient, Visit visit, Provider p
 		return encounter;
 	}
 	
+	/*
+	 * return String array with global property values
+	 */
+	protected String[] getArrayGlobalProperty(String globalPropertyName, boolean required) {
+		String globalProperty = administrationService.getGlobalProperty(globalPropertyName);
+		if (required && StringUtils.isEmpty(globalProperty)) {
+			throw new APIException("Configuration required: " + globalPropertyName);
+		}
+		return StringUtils.isEmpty(globalProperty) ? null : globalProperty.split(",");
```

Comment:
We can avoid needing to do null checks at all:

```suggestion
		return StringUtils.isEmpty(globalProperty) ? new String[0] : globalProperty.split(",");
```